.rubocop_todo.yml

```diff
@@ -1,15 +0,0 @@
-# This configuration was generated by
-# `rubocop --auto-gen-config`
-# on 2016-07-04 18:33:07 -0400 using RuboCop version 0.38.0.
-# The point is for the user to remove these configuration records
-# one by one as the offenses are removed from the code base.
-# Note that changes in the inspected code, or installation of new
-# versions of RuboCop, may require this file to be generated again.
-#
-
-Style/StringLiterals:
```

Comment:
We need these to enable StringLiteral checking. (On CI, Rubocop doesn't detect emoji in strings as being special characters, but seems to locally :/)